app/models/group.rb

```diff
@@ -750,7 +751,7 @@ def flair_type
   end
 
   def flair_url
-    flair_icon.presence || flair_upload&.url
+    flair_icon.presence || upload_cdn_path(flair_upload&.url)
```

Comment:
`upload_cdn_path(flair_upload&.url)` could explode if flair_upload is `nil`

Reply:
It did not in my tests:

```ruby
➜  discourse git:(master) rails c
Loading development environment (Rails 6.0.3.3)
[1] pry(main)> include GlobalPath
=> Object
[2] pry(main)> upload_cdn_path(nil)
=> nil
```

Reply:
It will when S3 uploads are enabled:

```rb
SiteSetting.s3_cdn_url = "https://test.com"
SiteSetting.s3_upload_bucket = "test"
SiteSetting.enable_s3_uploads = true
include GlobalPath
upload_cdn_path(nil)

NoMethodError: undefined method `[]' for nil:NilClass
from (…)/discourse/discourse/lib/file_store/s3_store.rb:169:in `cdn_url'
```

Reply:
How about now @CvX ?

Reply:
👌